blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DSetIO.java

```diff
@@ -58,4 +64,26 @@ protected BluefloodSetRollup fromByteBuffer(ByteBuffer byteBuffer) {
         return serDes.deserialize(byteBuffer);
     }
 
+    /**
+     * Retrieves the {@link BoundStatement} for a particular set metric
+     * @param metric
+     * @param granularity
+     * @return
+     */
+    @Override
+    protected BoundStatement getBoundStatementForMetric(IMetric metric, Granularity granularity) {
+        Object metricValue = metric.getMetricValue();
+        if (!(metricValue instanceof BluefloodSetRollup)) {
+            throw new InvalidDataException(
+                    String.format("getBoundStatementForMetric(locator=%s, granularity=%s): metric value %s is not type BfSetRollup",
```

Comment:
Should we write the actual type `BluefloodSetRollup` instead of `BfSetRollup` in the exception message? This comment applies to other places like counter, gauge etc as well.